autoload/ale/fixers/nixfmt.vim

```diff
@@ -0,0 +1,16 @@
+scriptencoding utf-8
+" Author: houstdav000 <houstdav000@gh0st.sh>
+" Description: Fix files with nixfmt
+
+call ale#Set('nix_nixfmt_executable', 'nixfmt')
+call ale#Set('nix_nixfmt_options', '')
+
+function! ale#fixers#nixfmt#Fix(buffer) abort
+    let l:executable = ale#Var(a:buffer, 'nix_nixfmt_executable')
+    let l:options = ale#Var(a:buffer, 'nix_nixfmt_options')
+
+    return {
```

Comment:
There is a `ale#Pad()` method you can use here:

```
'command': ale#Escape(l:executable) . ale#Pad(l:options)
```

see `:h ale#Pad()`

Reply:
Implemented in 78136f92.